articles/types/jsdoc.md

```diff
@@ -1,3 +1,179 @@
 # JsDoc support
 
-TODO
+Programs written in JavaScript and TypeScript can be annotated with (JsDoc comments)[http://usejsdoc.org/index.html]
+to give type information and quick-info descriptions of identifiers.
```

Comment:
quick-info is spelled quickinfo elsewhere

Reply:
changed